tool/get_localid

```diff
@@ -1,100 +1,100 @@
-#!/usr/bin/python
```

Comment:
Same for all executable files in tool directory: Cygwin doesn't support execution bit so all these files aren't changed. Ignore